.github/workflows/pr_tests.yml

```diff
@@ -124,51 +124,34 @@ jobs:
       #     sudo apt-get update
       #     sudo apt-get install libavdevice-dev libavfilter-dev libopus-dev libvpx-dev pkg-config
 
-      - name: Install PyTorch Linux and MacOS
-        if: startsWith(runner.os, 'Windows') != true
-        run: |
-          ./scripts/pytorch_install.sh ${{ matrix.torch-version }}
-
-      - name: Install PyTorch Windows
-        if: startsWith(runner.os, 'Windows')
+      - name: Adjust PyTorch versions
         run: |
-          ./scripts/pytorch_install.ps1 ${{ matrix.torch-version }}
+          python .github/adjust_torch_versions.py ./requirements.txt ${{ matrix.torch-version }}
+          cat ./requirements.txt
 
       - name: Install packages
         run: |
-          pip install -r requirements.txt --default-timeout=60
-          pip install -e . --default-timeout=60
+          echo "attempting to install"
+          cat ./requirements.txt
```

Comment:
```suggestion
```
this is not needed since you have cat in the step above

Reply:
That was for debugging, I was trying to figure out why it was failing. Turns out it was the Windows python install not matching the os.linesep and installing CUDA instead of CPU.